packages/v4/patternfly-docs/content/accessibility/badge/badge.md

```diff
@@ -3,30 +3,61 @@ id: Badge
 ---
 
-A **badge** is used to annotate other information like a label or an object name. Badges are typically used to reflect counts, like number of unread notifications. Badges are not interactive.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should not be able to focus on a badge.
+## Accessibility
 
-**Screen reader users** should be able to have a screen reader describe the contents of and purpose of a badge when it pops up, but not focus on it otherwise.
+To implement an accessible PatternFly **badge**:
 
-No props need to be added or modified for badge accessibility.
+- Ensure the badge itself is not interactive in anyway. 
+- Ensure that the surrounding context can convey to users via screen reader or similar assistive technologies the purpose of the badge.
+- Ensure badge information is not conveyed by only color, such as a read or unread badge.
 
-**As a caution**, badges can be styled using a variety of colors. Do not rely on color alone to communicate information 
-because it causes barriers to access for many readers. For example, colorblind and low vision users may not be able 
-to perceive the color differences, and screen readers do not announce colors to non-sighted readers.
+## Testing
 
-Since the badge doesn't get an accessible name and isn't focusable, each badge must have adequate contextual information 
-provided in the surrounding UI to convey the same information the color alone is conveying. A developer could consider
-using a label with an icon in it to supplement the color.
+ At a minimum, a badge should meet the following criteria:
 
-Note how difficult it is to determine that there are unread or actionable items associated with either of the badges
-in Figure 1 with its lack of visual cues. Compare Figure 1 to Figure 2 where there is an icon
-used to indicate that the second badge is associated with actionable or unread content.
+<List isPlain>
+  <ListItem>
+    <Checkbox id="badge-a11y-checkbox-1" label="Users should not be able to interact with or navigate to the badge with the keyboard, as it is not an interactive element." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="badge-a11y-checkbox-2" label="Users navigating via screen reader or similar assistive technology should understand the purpose of the badge by its surrounding context." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="badge-a11y-checkbox-3" label="Information about a badge is not conveyed by only color."/>
+  </ListItem>
+</List>
+
+## React customization
+
+A badge does not have any further React props for accessibility.
+
+## HTML/CSS customization
+
+A badge does not have any further HTML/CSS attributes or classes for accessibility.
+
+## Additional considerations
+
+Consumers must ensure they take any additional considerations when customizing a badge, using it in a way not described or recommended by PatternFly, or in various other specific use-cases not outlined elsewhere on this page.
```

Comment:
I don't think "use-case" should be hypenated.

Reply:
Good catch! Updating a couple of other files that hyphenated it ("use case" is used a lot more in other files in org)